docs/sources/enterprise/license-expiration.md

```diff
@@ -0,0 +1,38 @@
++++
+title = "License Expiration"
+description = ""
+keywords = ["grafana", "licensing"]
+type = "docs"
+[menu.docs]
+parent = "enterprise"
+weight = 8
++++
+
+# Expired License
```

Comment:
```suggestion
# Expired license
```

Reply:
Per the Documentation style guide, sentence case for all headings.

Reply:
https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md'